spotifyconnector/connector.py

```diff
@@ -445,6 +447,32 @@ def episodes(
 
             page += 1
 
+    def catalog(self) -> dict:
+        """Loads podcast catalog data.
+
+        Returns:
+            dict: [catalog]
+        """
+        url = self._build_url(
+            "user",
+            "shows"
+        )
+
+        end = dt.date.today()
+        start = end - dt.timedelta(days=30)
+
+        return self._request(
+            url,
+            params={
+                "page": 1,
+                "size": 200,
```

Comment:
What if we got more than 200 podcasts? 🤔 😉

Reply:
yup, that is a good question but didn't want to implement pagination or set the value too high. pagination would be the clean solution but not needed so far.

Reply:
Yeah, was just half-serious. 😉